### DISCUSSION TITLE
waterfall or grouped and stacked with opacity

### DISCUSSION THREAD

**Discussion by cwiese** [FormidableLabs/victory#2100] [General]
Is it possible to do this chart .. it is both grouped and stacked... and the first segment of the 2 - n groups are opacity = 0 ...aka the waterfall chart?

![image](https://user-images.githubusercontent.com/178527/63339651-5b981800-c313-11e9-8fcd-a0b499e8b50b.png)

**Comment by larissa-n** [comment]
Would be interested in seeing a solution for this. The y0 prop might be a cleaner way to create a waterfall chart than trying to work with an additional, hidden data series (not my idea - [saw it here](https://github.com/uber/react-vis/issues/1236#issuecomment-526948202)).
